Sources/OpenAPIURLSession/URLSessionTransport.swift

```diff
@@ -70,6 +70,31 @@ public struct URLSessionTransport: ClientTransport {
         /// - Parameter session: The URLSession used for performing HTTP operations.
         ///   If none is provided, the system uses the shared URLSession.
         public init(session: URLSession = .shared) { self.init(session: session, implementation: .platformDefault) }
+        /// Specifies the mode in which HTTP request and response bodies are processed.
+        public enum HTTPBodyProcessingMode {
```

Comment:
We generally do not use enums in public API because, right now, they are not extensible without breaking API. Instead for enum-like public API, we wrap an internal enum in a public struct with public static properties.